proto/osmosis/concentrated-liquidity/genesis.proto

```diff
@@ -51,6 +51,8 @@ message PoolData {
 message PositionData {
   Position position = 1;
   uint64 lock_id = 2 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
+  osmosis.accum.v1beta1.Record fee_accum_record = 3
+      [ (gogoproto.nullable) = false ];
```

Comment:
Are incentives going to be added separately?

Reply:
Incentives are being addressed by Sishir